client/app/migration/services/migration-request.service.js

```diff
@@ -0,0 +1,24 @@
+import { getStore } from '@zebrunner/core/store';
+import { getApplicationConfig } from '@zebrunner/core/store';
+
+export const MigrationRequestService = ($httpMock) => {
+    'ngInject'
+
+    return {
+        post: request('post'),
+        put: request('put'),
+        patch: request('patch'),
+        get: request('get'),
+        delete: request('delete'),
+    };
+
+    function getUrl() {
+        const state = getStore().getState();
+
+        return getApplicationConfig(state).api;
+    }
+
+    function request(method) {
+        return (url, ...params) => $httpMock[method](`${getUrl()}${url}`, ...params);
+    }
```

Comment:
`getURL` is too common, please rename to `getAPIUrl` (or even `getAPI` will be better)